extensions/wikia/WikiaPhotoGallery/WikiaPhotoGallery.class.php

```diff
@@ -118,6 +118,21 @@ function __construct() {
 			'hideoverflow' => false,
 			'sliderbar' => array('bottom','left')
 		);
+
+		$this->mAvalaibleUniqueParams = array_values(
```

Comment:
Also, should it be declared like https://github.com/Wikia/app/pull/6300/files#diff-6ddf00ef82821d11d4d1cffd5bd8a74cR56?